sites/public/src/components/listing/ListingViewSeeds.tsx

```diff
@@ -0,0 +1,752 @@
+        fallbackImageUrl={IMAGE_FALLBACK_URL}
+      />
+      <div className={`${styles["listing-main-details"]} seeds-m-bs-header`}>
+        <Heading
+          priority={1}
+          size={"xl"}
+          className={`${styles["listing-heading"]} seeds-m-be-text`}
+        >
+          {listing.name}
+        </Heading>
+        <div className={styles["listing-address"]}>
+          <div className={"seeds-m-ie-4"}>{oneLineAddress(listing.listingsBuildingAddress)}</div>
+          <div>
+            <Link href={googleMapsHref} newWindowTarget={true}>
+              {t("t.viewOnMap")}
+            </Link>
+          </div>
+        </div>
+
+        {listingTags?.length > 0 && (
+          <div className={`${styles["listing-tags"]} seeds-m-bs-3`}>
+            {listingTags.map((tag, index) => {
+              return (
+                <Tag variant={tag.variant} key={index}>
+                  {tag.title}
+                </Tag>
+              )
+            })}
+          </div>
+        )}
+
+        <p className={"seeds-m-bs-3"}>{listing.developer}</p>
+        <div className={`${styles["hide-desktop"]} seeds-m-b-3`}>{DueDate}</div>
+      </div>
+      <div className={styles["hide-desktop"]}>{ListingMainDetails}</div>
+    </div>
+  )
+
+  const RentSummary = (
+    <div className={styles["rent-summary"]}>
+      <div>
+        <Heading size={"lg"} className={"seeds-m-be-header"} priority={2}>
+          {t("t.rent")}
+        </Heading>
+        {amiValues.length > 1 &&
+          amiValues.map((percent) => {
+            const byAMI = listing.unitsSummarized.byAMI.find((item) => {
+              return parseInt(item.percent, 10) == percent
+            })
+
+            const groupedUnits = byAMI
+              ? getSummariesTable(byAMI.byUnitType, listing.reviewOrderType)
+              : []
+
+            return (
+              <React.Fragment key={percent}>
+                <Heading size={"md"} priority={3} className={"seeds-m-bs-content"}>
+                  {t("listings.percentAMIUnit", { percent: percent })}
+                </Heading>
+                <div className={"seeds-m-bs-header"}>
+                  <StandardTable
+                    headers={unitSummariesHeaders}
+                    data={groupedUnits}
+                    responsiveCollapse={true}
+                  />
+                </div>
+              </React.Fragment>
+            )
+          })}
+        {amiValues.length === 1 && (
+          <StandardTable
+            headers={unitSummariesHeaders}
+            data={getSummariesTable(
+              listing.unitsSummarized.byUnitTypeAndRent,
+              listing.reviewOrderType
+            )}
+            responsiveCollapse={true}
+          />
+        )}
+      </div>
+    </div>
+  )
+
+  const EligibilitySection = (
+    <CollapsibleSection
+      title={t("listings.sections.eligibilityTitle")}
+      subtitle={t("listings.sections.eligibilitySubtitle")}
+      priority={2}
```

Comment:
question: this might be more in the nitty gritty of the design than what you were looking for, but I noticed that on mobile this Eligibility (and all subsequent) H2 headers are bigger than they are in the designs for Mobile. Looks like the figma has it at 20px whereas this is displaying it as 26px. The desktop view is correct at 26px. Is that intended?

Reply:
Yeah I'm working w design on heading tokens specifically. I think updating the colors and sizes is a later pass that would start in Seeds, I'm hoping for another update from Jesse today. What's in the design for heading tokens is new and isn't currently supported by default in Seeds but hopefully soon! It would have impacts across the whole app.

Reply:
That makes total sense! Yeah, no need to update it as part of this PR